### ISSUE TITLE
Support for the Linux plutil ("plistutil" on Debian)

### ISSUE THREAD

**Issue by knghtbrd** [darfink/vim-plist#1]
Hi, wondering if you'd be willing to add support for plistutil written by the cross-platform libmobiledevice.org people:

https://github.com/libimobiledevice/libplist

(plistutil lives in the tools subdir)

It only supports XML and binary formats at the moment and its syntax is very subtly different from Apple's plutil (probably why it was renamed.)

How often am I working on plists on Linux? Just about often enough to be annoyed when I have to convert them myself. I was just about to start writing a script called plutil to massage the command line when it occurred to me that changing the plugin was the solution that benefited more people.

**Comment by darfink** [comment]
Implemented in https://github.com/darfink/vim-plist/commit/0d8a2aa3e332cdc8aa70c53793abf7e149451bc3

**Comment by tomasreimers** [comment]
Hi! I ran across this thread while Googling for the same thing myself. After looking at a few solutions for my own company ([Screenplay](https://screenplay.dev/)) I decided to fork and iterate on a open-source, cross-platform, drop-in replacement for `plutil`:

https://github.com/screenplaydev/plutil 

It's forked from Facebook's xcbuild (a tool developed by them to build xcode projects on Linux), but stripped down to just provide plist-editting functionality. That way you won't need to maintain separate code-paths for Mac and Linux environments.

Hope that's helpful!
